### ISSUE TITLE
Bootloop after installing cm71-111109.1032-NIGHTLY-Defy

### ISSUE THREAD

**Issue by elguaxo** [CyanogenDefy/android_device_motorola_jordan#146]
After upgrading from cm71-111106.0217-NIGHTLY-Defy to cm71-111109.1032-NIGHTLY-Defy I got a bootloop and I had to pull out the battery. It's the first time this kind of problem happens to me. For the records, I wiped the cache and dalvik cache before and after installing this nightly.

**Comment by elguaxo** [comment]
The problem seems to be solved on cm71-111110.0453-NIGHTLY-Defy

**Comment by charlie2alpha** [comment]
Yes, but as Tanguy (Epsylon3) said himself in the forums, it's not really solved yet. The next nightly will have more logging ability to debug boot issues, so we may finally learn what causes the bootloop.

**Comment by elguaxo** [comment]
FWIW. I had no problems upgrading to cm71-111110.0453-NIGHTLY-Defy. The whole upgrade to update-cm-7.1.0-11-Defy-signed went fine too. Then just for doing a test I went to the CWM menu one more time and tried to reboot and got the bootloop, somewhere during/after the button lights flickering. Then I did another upgrade to cm71-111112.0846-NIGHTLY-Defy and it worked and after that upgrade I made the same test -> CWM menu -> reboot and I got a bootloop again.
